packages/business-rules/src/rules/appeal/deadline-period.js

```diff
@@ -1,22 +1,39 @@
 const { appeal } = require('../../config');
 const isValid = require('../../validation/appeal/type/is-valid');
+const isValidApplicationDecision = require('../../validation/appeal/application-decision/is-valid');
 const { APPEAL_ID } = require('../../constants');
 const BusinessRulesError = require('../../lib/business-rules-error');
 
 /**
  * @description Determine the appeal's deadline period.
  *
- * @param {Number} expiryPeriodInWeeks a positive number
+ * @param {string} appealType
+ * @param {string} applicationDecision
+ *
+ * @return {string} appeal deadling duration and time
  *
  * @throws {BusinessRulesError}
  */
-module.exports = (appealType = APPEAL_ID.HOUSEHOLDER) => {
+module.exports = (appealType = APPEAL_ID.HOUSEHOLDER, applicationDecision) => {
```

Comment:
It might be that in the future we need to use a different deadline date based on the value of a different field in the form so if `applicationDecision` had a more generic name then this value wouldn't necessarily be tied to this field.

Reply:
Note sure if its appropriate to change the name as this value is being validated against list of application decision in line 22 of this file.

Reply:
Ok fair enough, I'm not so keen on this being tied to the application decision field so much but it's fine for now.